### ISSUE TITLE
Terraria 1.4

### ISSUE THREAD

**Issue by jbro129** [Fe7n/TShockMobile#30]
Do you plan on updating this to support Terraria mobile 1.4?

**Comment by Fe7n** [comment]
No, try the 1.4 pc version, it might work. If not, just wait for official xplatform support ;)

**Comment by EMREOYUN** [comment]
1.4 have couple issues with Bouncer and it's still in beta :/
